src/pages/mpl-404/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Overview
+metaTitle: MPL-404 - Overview
+description: Provides a high-level overview of the framework and on-chain protocol for hybrid assets.
+---
+
+MPL-404, is a new model for digital assets, web3 games, and on-chain communities. At the core of the model is a swap program that trades a fixed number of fungible assets for a non-fungible asset and vice versa. The swap is a dual escrow system, ensuring that all available non-fungible assets are backed by escrowed fungibles and vice versa.
+
+The ability to freely move between fungible and non-fungible assets in a predictable way allows this new asset class (often called 404s) to take advantage of the best aspects of both asset types. Non-fungible assets can now benefit from the liquidity, distribution, and DeFi opportunities associated with fungible assets. Conversely, fungible assets can now benefit from the improved utility, collectability, and identity that come from the non-fungible world.
+
+MPL-404 also includes the option to "re-roll" the asset each time its swapped. For example, every non-fungible asset can have its metadata blanked as it enters the escrow wallet and randomly reassigned (rerolled) as it leaves escrow. The creator has the ability to both manage available traits as well as to charge a small fee during the swap (typically when swapping into an NFT). MPL-404 can add inherit “loot box” gamification to every 404 project and can serve as an alternative source of revenue (e.g. unique limited time traits for an NFT community or randomized in-game rewards). It also offers the ability to craft more dynamic collections that can evolve to better suit the needs of the project and community over time.
```

Comment:
Nit: At the beginning it says "MPL-404 also includes..." and at the end it says "It also offers..."

I suggest taking maybe the first "also" out as there's too many "also"s imo.